src/iperf_api.c

```diff
@@ -804,25 +819,26 @@ iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
         {"nstreams", required_argument, NULL, OPT_NUMSTREAMS},
         {"xbind", required_argument, NULL, 'X'},
 #endif
-	{"pidfile", required_argument, NULL, 'I'},
```

Comment:
you should not touch other code formatting. please check your editor (probably autoformat was applied).
such changes are unrelated to our pull request.